montage/admin_endpoints.py

```diff
@@ -606,13 +606,9 @@ def finalize_campaign(user_dao, campaign_id):
     coord_dao = CoordinatorDAO.from_campaign(user_dao, campaign_id)
     last_rnd = coord_dao.campaign.active_round
 
-    if not last_rnd:
-        raise InvalidAction('no active rounds')
-
-    if last_rnd.vote_method != 'ranking':
-        raise InvalidAction('only ranking rounds can be finalized')
-
-    campaign_summary = coord_dao.finalize_ranking_round(last_rnd.id)
+    campaign_summary = None
```

Comment:
Should there be a different status for a campaign that's done but doesn't have a summary?